### PR TITLE
feat: dispatch event to `flarum/installation-packages` on release

### LINKED ISSUE #3561: Automated workflow to generate installable flarum packages

### PR DESCRIPTION
**Pull request by SychO9**
**Changes proposed in this pull request:**
Auto trigger build of installation packages on [flarum/installation-packages](https://github.com/flarum/installation-packages).

* Needs a `PACKAGES_BUILD_TOKEN` secret added first. The token must have permission to trigger the `workfow_dispatch` event on the target repository.
* Then needs to be tested.

**Necessity**

- [ ] Has the problem that is being solved here been clearly explained?
- [ ] If applicable, have various options for solving this problem been considered?
- [ ] For core PRs, does this need to be in core, or could it be in an extension?
- [ ] Are we willing to maintain this for years / potentially forever?

**Confirmed**

- [ ] Frontend changes: tested on a local Flarum installation.
- [ ] Backend changes: tests are green (run `composer test`).
- [ ] Core developer confirmed locally this works as intended.
- [ ] Tests have been added, or are not appropriate here.

### CODE REVIEW

**Comment by luceos** [comment]
Is the idea behind your workflow to run after the release, cq the version has been tagged? Because the prepare release workflow only gathers changes, commits, contributors and donators. It doesn't actually tag (yet) because tags need changelogs..

**Comment by SychO9** [comment]
Argh.. true, you run the `prepare release` workflow first, then tag.

That's fine we can always trigger the tarball build manually as well.

**Comment by SychO9** [comment]
Actually, let me just switch to a separate workflow that runs on release

File: .github/workflows/build-install-packages.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: Build Install Packages
+
+on:
+  release:
+    types: [released]
+
+env:
+  VERSION: ${{ github.event.release.tag_name }}
+  PHP_VERSIONS: '7.3 7.4 8.0 8.1'
+  INSTALL_PACKAGES_INPUTS: '{ "flarum_version": "{0}", "php_versions": "{1}" }'
```
Comment (askvortsov1) [inline .github/workflows/build-install-packages.yml:10]: Where are these magic variables coming from? Is there no named equivalent?

File: .github/workflows/build-install-packages.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+name: Build Install Packages
+
+on:
+  release:
+    types: [released]
+
+env:
+  VERSION: ${{ github.event.release.tag_name }}
+  PHP_VERSIONS: '7.3 7.4 8.0 8.1'
```
Comment (askvortsov1) [inline .github/workflows/build-install-packages.yml]: Can we use a list?

File: .github/workflows/build-install-packages.yml (hunk 1 above)
Comment (SychO9) [reply .github/workflows/build-install-packages.yml:10]: it's the exact values of the env variables above it.

File: .github/workflows/build-install-packages.yml (hunk 2 above)
Comment (SychO9) [reply .github/workflows/build-install-packages.yml]: the value can only be string, integer or boolean, and I don't think there's benefit to declaring it as a JSON array only to reconvert it to a bash list.

Comment (SychO9) [inline .github/workflows/build-install-packages.yml]:
```suggestion
  PHP_VERSIONS: '8.1 8.2 8.3'
```

**Pull request merged**